lib/studygroup_web/live/page_live.html.leex

```diff
@@ -6,6 +6,10 @@
     <p class="text-lg md:text-xl text-gray-600 mb-6">
       Wanna join? If yes, please, unite with us at our Slack community <a href="">here now </a>:)
     </p>
+    <p class="text-lg md:text-xl text-gray-600 mb-6">
+      See our resources so far: <br> <a href="/students">Our Students</a> | <a href="/group"> Our Groups</a>
```

Comment:
@ykostov @N-Patarov, let's use Phoenix' convention for creating links here, please?